src/extensions/default/bramble/lib/ConsoleInterfaceManager.js

```diff
@@ -75,20 +78,28 @@ define(function (require, exports, module) {
 
         if(!item.length) {
           item = Strings.CONSOLE_EMPTY_STRING;
-          $element.addClass("empty-string");
+          $lineElement.find(".log-entry").addClass("empty-string");
         }
 
-        $element.text(item);
-
+        $lineEntry.text(item);
+        
         if(panel.isVisible()) {
-          $element.addClass("new-log");
+            $lineElement.find(".log-entry").addClass("new-log");
+        }
+
+        //Add a line number to the new entry
+        if(linesEl.find(".line").length === 0){
+            $lineNumber.html(1);
+        }else{
+            var lastLineNum = linesEl.children().last().children(".linenumber").html();  
```

Comment:
Instead of this, let's keep an internal integer variable that you just increment, and reset when we clear.  Reading it out of the DOM like this is fragile.